### PR TITLE
fixed branch refactoring

### PR DESCRIPTION
**Pull request by RexBrown122**

### CODE REVIEW

File: src/app/components/planet-details/planet-details.component.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div id="content">
+    <div id="Planets-List" class="row itemBlock">
+        <div class="itemContainer col-md-3 col-sm-6 col-12" *ngFor="let planet of planets" [planet] = "planet">
+            <a id="select-ep">{{planet.name}}</a>
```
Comment (mosdevly) [inline src/app/components/planet-details/planet-details.component.html]: What is `select-ep`?

File: src/app/components/planet-details/planet-details.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    fixture = TestBed.createComponent(PlanetDetailsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  // it('should render the h1 as Planets in a Galexy far far away...', () => {
```
Comment (mosdevly) [inline src/app/components/planet-details/planet-details.component.spec.ts]: If its commented it out, YAGNI. https://en.wikipedia.org/wiki/You_aren't_gonna_need_it

Just a thought :)

File: src/app/components/planets-list/planets-list.component.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div class="container text-center">
-    <h1 id="title">Planets in a Galexy far far away...</h1>
+    <h1 id="title">Planets in a Galaxy far far away...</h1>
 </div>
-<div id="content">
-
+<div>
+    <app-planet *ngFor="let planet of planets" [planet]="planet"></app-planet>
```
Comment (mosdevly) [inline src/app/components/planets-list/planets-list.component.html:5]: Hitting your stride!

File: src/app/models/planets.spec.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { Planets } from './planets';
+import { Planets } from './Planets';
```
Comment (mosdevly) [inline src/app/models/planets.spec.ts]: The class naming convention doesn't apply directly to filenames. This is why we've been able to use things like `counter-list.componenet.ts` and inside the file the class is named `CounterListComponent`.

**Comment by 100xlongx** [comment]
planet imports are failing on this one too

**Review by mosdevly — COMMENTED** [review]
Very clean and looks good so far. make some minor tweaks and we can get this merged! See comments below.

File: src/app/models/planets.spec.ts (hunk 4 above)
Comment (RexBrown122) [reply src/app/models/planets.spec.ts]: Do you mean that the file path name for Planets shouldn't be capitalized even though the file names are capitalized?

**Comment by RexBrown122** [comment]
> planet imports are failing on this one too

yeah i have been trying many ways to fix this. I'm not receiving errors locally, but apparently Git doesn't like it.

File: src/app/models/planets.spec.ts (hunk 4 above)
Comment (mosdevly) [reply src/app/models/planets.spec.ts]: Correct. The file names can remain the same way we've used them so far.

**Comment by mosdevly** [comment]
Closing this to retry with a different PR. Looks like github is stalled.

**Pull request closed without merging**